pygenomeviz/0.3.2/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:jammy as app
+
+ARG PYGENOMEVIZ_VER="0.2.2"
```

Comment:
Should this be 0.3.2?

Reply:
Yes! It's likely the most important variable!!!!